Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -238,3 +241,11 @@ endif()
 if(USE_DISCORD_PRESENCE)
   target_compile_definitions(dolphin-emu PRIVATE -DUSE_DISCORD_PRESENCE)
 endif()
+
+add_executable(bbaserver BBAServer_main.cpp)
+
+target_link_libraries(bbaserver Qt5::Core Qt5::Network bbaserverlib)
+
+add_library(bbaserverlib BBAClient.cpp BBADebug.cpp BBAServer.cpp)
```

Comment:
The lib should likely be introduced before the executable that uses it above.

Reply:
moving it above dolphin-emu?